src/app.js

```diff
@@ -1 +1,126 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+async function fetchData() {
+    try {
+        const response = await axios.get('https://restcountries.com/v2/all');
+        const countries = response.data
+        // console.log(response.data);
+
+
+        countries.sort((a, b) => {
+            return a.population - b.population;
+        });
```

Comment:
Je zou dit nog iets korter kunnen schrijven
```suggestion
        countries.sort((a, b) => a.population - b.population);
```